proto/execution_stats.proto

```diff
@@ -114,4 +114,6 @@ message GetExecutionResponse {
   context.ResponseContext response_context = 1;
 
   repeated Execution execution = 2;
+
+  acl.ACL acl = 3;
```

Comment:
I think this belongs on the item itself -- like on the Execution or Invocation.

Reply:
I wasn't actually sure about this myself -- will it be the case that executions can have differing permissions in a single `GetExecutionResponse`? Or are you thinking more in the organizational sense -- that executions should have the permissions, not the RPC response?

Reply:
I think both - executions/invocations will likely have the same permission, but it'd be possible for some to be readable by others and others not be. Organizationally it also makes sense to be on the invocation/execution - because that's where the level at which the permissions are stored.

Reply:
Makes sense, done!

Reply:
Yeah, can we move this inside the Execution message?

Search is a good example -- there might be things returned that are readable but not writeable.